docs/deployment/landing_page.rst

```diff
@@ -249,3 +249,77 @@ that user and group file permissions are locked down and that modules or
 gateway interfaces for dynamic scripting languages are not enabled. You
 don't want any unnecessary code or services running as this increases
 the attack surface.
+
+Landing page content suggestions
+--------------------------------
+
+The content below presents sample text for the SecureDrop component of a news 
+organization’s tips page. It does not account for any specific legal 
+or organizational needs, but should provide guidance for any outlet getting 
+started on crafting landing page language. Any tweaks to the sample content 
+should be left to the legal and editorial discretion of the individual outlet, 
+and should be viewed as essential to upholding source protection and transparency.
```

Comment:
by "transparency" do you mean "anonymity" ? Maybe it's equivalent in English but ... I'm not a native English speaker ;-)

Reply:
The use of "transparency" is meant to indicate that the news organization is openly and publicly stating the protections it strives to provide for sources, including, yes, anonymity. As such I'm OK with the phrasing here, but I acknowledge it could be made clearer over time.